### PR TITLE
NSFS | NC | set /etc/noobaa.conf.d as default config_root and a small bug fix

### LINKED ISSUE #7495: Need to support input path as a filesystem for bucket creation
**Is your feature request related to a problem? Please describe.**
Currently we have observed that noobaa rpm(as nsfs service) has fixed path of filesystem, where each time buckets are created. But as a user, dynamically path can be given during bucket creation.

Example of nsfs.service
*************************
```
[Unit]
Description=The NooBaa nsfs service.

[Service]
Restart=always
RestartSec=2
User=root
Group=root
Environment=
ExecStart=/usr/local/node/bin/node /usr/local/noobaa-core/src/cmd/nsfs.js /tmp/test/
ExecStop=/bin/kill QUIT $MAINPID
WorkingDirectory=/usr/local/noobaa-core/
LimitNOFILE=65536

[Install]
```

**Describe the solution you'd like**
Description of what you want to happen.

**Describe alternatives you've considered**
Description of alternative solutions or features you've considered.

**Additional context**

### PR DESCRIPTION
**Pull request by romayalon**
### Explain the changes
1. nsfs.js -
   1.1. set /etc/noobaa.conf.d as default config_root 
   1.2. add --simple flag for running simple bucketspace
2. nsfs.service -  remove config_root flag 
3. bucketspace_fs - small bug fix, on create bucket - check if the json config file exists and throw BUCKET_ALREADY_EXISTS. The bug was that it was checked only on the mkdir. 
### Issues: Fixed #xxx / Gap #xxx
1. Fixed #7495 

### Testing Instructions:
1. 

- [ ] Doc added/updated
- [ ] Tests added

### CODE REVIEW

File: src/cmd/nsfs.js [hunk 1]
```diff
@@ -220,7 +221,11 @@ async function main(argv = minimist(process.argv.slice(2))) {
         const gid = Number(argv.gid) || process.getgid();
         const access_key = argv.access_key && new SensitiveString(String(argv.access_key));
         const secret_key = argv.secret_key && new SensitiveString(String(argv.secret_key));
-        const config_root = argv.config_root ? String(argv.config_root) : '';
+        const simple_mode = Boolean(argv.simple) || false;
```
Comment (tangledbytes) [inline src/cmd/nsfs.js:224]: Do we need `|| false` here?

Comment (romayalon) [reply src/cmd/nsfs.js]: I removed it, thanks!

File: src/cmd/nsfs.js [hunk 2]
```diff
@@ -62,18 +62,19 @@ Options:
...
     ## multi user mode
 
-    --iam_json_schema                           Print the json schema of the identity files in iam dir.
-    --iam_ttl <seconds>     (default 60)        Identities expire after this amount of time, and re-read from the FS.
-    --config_root <dir>     (default none)      Configuration files for Noobaa standalon NSFS. It includes config files for environment variables(<config_root>/.env), 
-                                                local configuration(<config_root>/config-local.js), authentication (<config_root>/accounts/<access-key>.json) and 
-                                                bucket schema (<config_root>/buckets/<bucket-name>.json).
+    --iam_json_schema                                       Print the json schema of the identity files in iam dir.
+    --iam_ttl <seconds>     (default 60)                    Identities expire after this amount of time, and re-read from the FS.
+    --config_root <dir>     (default /etc/noobaa.conf.d)    Configuration files for Noobaa standalon NSFS. It includes config files for environment variables(<config_root>/.env),
```
Comment (tangledbytes) [inline src/cmd/nsfs.js:75]: I think we might need this
```suggestion
    --config_root <dir>     (default ${config.NSFS_NC_DEFAULT_CONF_DIR})    Configuration files for Noobaa standalon NSFS. It includes config files for environment variables(<config_root>/.env), 
```

Comment (romayalon) [reply src/cmd/nsfs.js]: done

**Review by tangledbytes — APPROVED** [review]

**Pull request merged**